clients/client-go/tcworkermanager/types.go

```diff
@@ -33,6 +33,10 @@ type (
 		Token string `json:"token"`
 	}
 
+	// Provider-specific information
+	ProviderData struct {
+	}
```

Comment:
This looks like it might not be what you want.

Reply:
It is what I want -- at the moment, no providers support any fields, so nothing is allowed.  In the next PR, when I add static-provider support, it will be an anyOf, similar to 

https://github.com/taskcluster/taskcluster/blob/8bf75678626567ec2fdae3889dff2f6d86235958/services/worker-manager/schemas/v1/register-worker-request.yml#L15-L32

(which gets treated as a `json.RawMessage`)

Reply:
Ah cool, I thought maybe it was a freeform blob, but that makes sense.

Note, that `anyOf` should be `oneOf`.